ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/cloudinit/DefaultCCMV1KeyRemapper.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.cloudbreak.ccmimpl.cloudinit;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.cloudera.thunderhead.service.minasshdmanagement.MinaSshdManagementProto.MinaSshdService;
+import com.cloudera.thunderhead.service.minasshdmanagement.MinaSshdManagementProto.PublicKey;
+import com.sequenceiq.cloudbreak.ccm.cloudinit.CCMV1KeyRemapper;
+import com.sequenceiq.cloudbreak.ccm.exception.CcmException;
+import com.sequenceiq.cloudbreak.ccmimpl.altus.GrpcMinaSshdManagementClient;
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+
+@Component("DefaultCCMV1KeyRemapper")
```

Comment:
So the reason I have this is because I have this magic injecting the right overriding class in one of the Stack classes:

```
@Inject
    @Qualifier("DefaultCCMV1KeyRemapper")
    private CCMV1KeyRemapper ccmV1KeyRemapper;
```

The `DefaultCCMV1KeyRemapper` is not visible from cloudbreak core due to not being wired in via Maven, so I think this allows it to be injected. I simply copied this idea from other uses of this. It might work without the component naming but it would take me too long to confirm.